DMCompiler/DM/DMExpression.cs

```diff
@@ -195,7 +203,8 @@ sealed class ArgumentList {
 
         DMComplexValueType paramType = param.ExplicitValueType ?? DMValueType.Anything;
 
-        if (!expr.ValType.IsAnything && !paramType.MatchesType(expr.ValType)) {
+
+        if ((!expr.ValType.IsAnything && !paramType.MatchesType(expr.ValType))) {
```

Comment:
Unnecessary empty line & parenthesis
```suggestion
        if (!expr.ValType.IsAnything && !paramType.MatchesType(expr.ValType)) {
```